### DISCUSSION TITLE
Παραδοτέο 2 οδηγίες για γραμμή εντολών και ομάδες

### DISCUSSION THREAD

**Discussion by ChristosMpazdanhs** [courses-ionio/sw#1147] [Q&A] [answered]
Που μπορώ να δω οδηγίες σχετικά με το παραδοτέο 2 (Γραμμή εντολών (systemd) και διαδικασία συνεργασίας με pull request στον οργανισμό της ομάδας σας);

**Comment by epidrome** [comment, marked as answer]
οι οδηγίες βρίσκονται στους αντίστοιχους συνδέσμους που προστέθηκαν τώρα, ευχαριστώ για την παρατήρηση!
